app/templates/render-tree.hbs

```diff
@@ -1,3 +1,7 @@
+{{#unless isClosed}}
```

Comment:
As mentioned above, you already have the conditional, so instead of removing the DOM element you should mutate the property.